### PR TITLE
make nasa swagger skin work here

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
Brief summary of changes if not sufficiently described by commit messages.

## ⚙️ Test Data and/or Report
One of the following should be included here:
* Reference to regression test included in code (preferred wherever reasonable)
* Attach test data here + outputs of tests

## ♻️ Related Issues

- fixes NASA-PDS/pds-api#234
- fixes https://github.com/NASA-PDS/pds-api/issues/196 

I did not find where to get the version of the application from, the only place where it is is in the maven project pom.xml, I believe we are loosing that when we launch the application inside docker with the java command.

### CODE REVIEW

**Review by nutjob4life — APPROVED** [review]
Looks fine!

Just a brief request:

> Brief summary of changes if not sufficiently described by commit messages.
>
> ⚙️ Test Data and/or Report
> 
> One of the following should be included here:
> 
> Reference to regression test included in code (preferred wherever reasonable)
> Attach test data here + outputs of tests

Is there any useful text to fill in here? Should the template be altered? If not, I would go ahead and delete this text 😄

File: docker/.env [hunk 1]
```diff
@@ -22,7 +22,8 @@ ES_DISCOVERY_TYPE=single-node
 # --------------------------------------------------------------------
 
 # Docker image of the Registry API
-REG_API_IMAGE=nasapds/registry-api-service:1.1.10
+#REG_API_IMAGE=nasapds/registry-api-service:1.1.10
+REG_API_IMAGE=nasapds/registry-api-service:latest
```
Comment (ramesh-maddegoda) [inline docker/.env:26]: @tloubrieu-jpl , Is this change intentional? I understand that we had some issues with some versions of Registry API. However, is it possible to have an exact working Registry-API version here instead of latest version?

Comment (tloubrieu-jpl) [reply docker/.env:25]: @ramesh-maddegoda yes that was intentional, for the main branch HEAD I was thinking to have the latest images and for the tagged version the tagged images. At least that is how I envisioned the usage of this docker compose.

To have the latest images is useful for development/integration testing purpose.

When I'll do a tag for registry 1.1.3, later today, I'll switch to tagged version for all the images.
